backend/src/nodes/nodes/image_channel/transparency_split.py

```diff
@@ -39,14 +39,19 @@ def __init__(self):
 
     def run(self, img: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
         """Split a multi-channel image into separate channels"""
+        logger.info(img.shape)
 
         if img.ndim == 2:
             logger.debug("Expanding image channels")
             img = np.tile(np.expand_dims(img, axis=2), (1, 1, min(4, 3)))
         # Pad with solid alpha channel if needed (i.e three channel image)
-        elif img.shape[2] == 3:
+        if img.shape[2] == 3:
             logger.debug("Expanding image channels")
             img = np.dstack((img, np.full(img.shape[:-1], 1.0)))
+        # Convert from gray to RGB then pad with solid alpha channel
+        if img.shape[2] == 1:
+            logger.debug("Expanding image channels")
+            img = np.dstack((img, img, img, np.full(img.shape[:-1], 1.0)))
```

Comment:
Why not just do `img = as_target_channels(img, 4)`?

Reply:
Because I forgot that existed